### ISSUE TITLE
Hosted Worlds: can't export save

### ISSUE THREAD

**Issue by beastyn** [StrangeLoopGames/EcoIssues#16731]
1) Create a new account
2) Buy a world
3) Wait till its loaded
4) Press Export Save in the Advanced Options
5) Get and error "There is no world with such ID"
![bandicam 2020-06-17 17-05-43-898](https://user-images.githubusercontent.com/27898520/84896857-2046b100-b0be-11ea-8f65-4acc6790802a.jpg)

**Comment by rainfarmer** [comment]
Also, cannot see game versions. im guessing its same bug

**Comment by denysaw** [comment]
@beastyn Current error is already not `There's no world with such ID`, but `You are not logged it`. It's cuzz your new user doesn't own Eco. And he shouldn't, but for now `authtoken` (which needs for world api call) has only that user, which owns Eco. So this issue will be solved only after this one implemented: https://github.com/StrangeLoopGames/EcoIssues/issues/16856
So this issue will be postponed till issue above will be resolved. Usually there's Postpone label for that or `Waiting`, but I didn't find such. Leaving as is.

**Comment by denysaw** [comment]
Label: Status: Postponed/Waiting (for related changes)

**Comment by j3fffff** [comment]
staging auth stuff all updated. thats working now. 
but now, on staging site and staging cw, getting this error on export:


`<Error> <Code>PermanentRedirect</Code> <Message>The bucket you are attempting to access must be addressed using the specified endpoint. Please send all future requests to this endpoint.</Message> <Endpoint>hosted-worlds-saves.s3.amazonaws.com</Endpoint> <Bucket>hosted-worlds-saves</Bucket> <RequestId>5F642DC5E541A30C</RequestId> <HostId>j18ru5TBHHY6HqzNJe5+01JhpT4RM3xatPEstcM5bAzJPvghxfWAA4i4RP4tT2lSf4f614ANmvU=</HostId> </Error>`

**Comment by denysaw** [comment]
Done. Test on staging.

**Comment by beastyn** [comment]
@denysaw should it work properly on developland?

**Comment by denysaw** [comment]
@beastyn Nope, only staging.
